docs/fastdds/dds_layer/core/waitsets/waitsets.rst

```diff
@@ -11,20 +11,21 @@ to notify communication status changes (including arrival of data) to the applic
 
 This mechanism is wait-based. Its general use pattern is as follows:
 
-* The application indicates which relevant information it wants to get, by means of Condition
-  objects (GuardCondition, StatusCondition, or ReadCondition) and attaching them to a wait-set.
-* It then waits on that wait-set until the trigger value of one or several Condition objects become
-  true.
-* It then uses the result of the wait (i.e., the list of Condition objects with
+* The application indicates which relevant information it wants to get, by means of :ref:`api_pim_condition`
+  objects (:ref:`api_pim_guardcondition`, :ref:`api_pim_statuscondition`, or ReadCondition)
+  and attaching them to a :ref:`api_pim_waitset`.
```

Comment:
Maybe we can include a reference to the API: `WaitSet::attach_condition`. Also, I suggest including the `WaitSet::detach_condition` API to let the user know that a condition can be detached when it is no longer of interest.